api/src/test/java/org/openmrs/module/metadatadeploy/MetadataUtilsTest.java

```diff
@@ -189,7 +189,7 @@ public void getForm_shouldThrowExceptionForNonExistent() {
 	@Test
 	public void getLocation_shouldFetchByUuid() {
 		Location unknown = Context.getLocationService().getLocation(1);
-		Assert.assertThat(MetadataUtils.getLocation("dc5c1fcc-0459-4201-bf70-0b90535ba362"), is(unknown));
```

Comment:
Why did you change this?

Reply:
There was an exception thrown that was `org.openmrs.module.metadatadeploy.MissingMetadataException: No such Location with identifier 'dc5c1fcc-0459-4201-bf70-0b90535ba362'` so having that changed to `8d6c993e-c2cc-11de-8d13-0010c6dffd0f` with reference to the location UUID in [this wiki page](https://wiki.openmrs.org/display/docs/UUIDs) did the trick

Reply:
Does that happen only with your changes?

Reply:
Yes, that's right